### PR TITLE
Fixing error messages

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
Fix error messages - made it more readable.

### How to test this PR?
Run e2e tests.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Comment by Katka92** [comment]
[crw-ci-test --rebuild]

File: tests/e2e/pageobjects/ide/Ide.ts [hunk 1]
```diff
@@ -49,10 +49,15 @@ export class Ide {
                 Logger.warn('StaleElementException occured during waiting for IDE. Sleeping for 2 secs and retrying.');
                 this.driverHelper.wait(2000);
                 await this.driverHelper.waitAndSwitchToFrame(By.css(Ide.IDE_IFRAME_CSS), timeout);
```
Comment (Ohrimenko1988) [inline tests/e2e/pageobjects/ide/Ide.ts:51]: looks like you missed 
```
return;
```
here

File: tests/e2e/pageobjects/ide/Ide.ts [hunk 2]
```diff
@@ -74,9 +79,10 @@ export class Ide {
             await this.driverHelper.waitVisibility(exitCodeNotificationLocator, timeout);
         } catch (err) {
             if (err instanceof error.TimeoutError) {
-                Logger.error(`Ide.waitTaskExitCodeNotificationBoolean wait for notification timed out.`);
+                Logger.error(`Ide.waitTaskExitCodeNotificationBoolean wait for notification timed out after ${timeout}.`);
+                throw err;
             } else {
```
Comment (ScrewTSW) [inline tests/e2e/pageobjects/ide/Ide.ts]: Unnecessary `else`

Comment (Ohrimenko1988) [inline tests/e2e/pageobjects/ide/Ide.ts]: Looks like "else" redundant here
```
if (err instanceof error.TimeoutError) {
    Logger.error(`Ide.waitTaskExitCodeNotificationBoolean wait for notification timed out after ${timeout}.`);
    throw err;
}

Logger.error(`Waiting for task notification failed.`)
throw err;
```

File: tests/e2e/pageobjects/ide/Ide.ts [hunk 3]
```diff
@@ -165,8 +171,14 @@ export class Ide {
             } catch (err) {
                 if (err instanceof error.NoSuchWindowError) {
                     await this.driverHelper.waitVisibility(idePartLocator, timeout);
+                }
```
Comment (Ohrimenko1988) [inline tests/e2e/pageobjects/ide/Ide.ts:174]: Looks like  ```return;``` missed here

Comment (Ohrimenko1988) [inline tests/e2e/pageobjects/ide/Ide.ts:174]: Looks like  ```return;``` missed here

File: tests/e2e/pageobjects/ide/Ide.ts [hunk 4]
```diff
@@ -165,8 +171,14 @@ export class Ide {
             } catch (err) {
                 if (err instanceof error.NoSuchWindowError) {
                     await this.driverHelper.waitVisibility(idePartLocator, timeout);
```
Comment (Ohrimenko1988) [inline tests/e2e/pageobjects/ide/Ide.ts:173]: Looks like  ```return;``` missed here

**Review by Ohrimenko1988 — APPROVED** [review]

**Pull request merged**